loader/db.go

```diff
@@ -34,213 +36,151 @@ import (
 
 // Conn represents a live DB connection
 type Conn struct {
-	cfg *config.SubTaskConfig
-
-	db *sql.DB
+	cfg      *config.SubTaskConfig
+	baseConn *baseconn.BaseConn
 }
 
-func (conn *Conn) querySQL(ctx *tcontext.Context, query string, maxRetry int, args ...interface{}) (*sql.Rows, error) {
-	if conn == nil || conn.db == nil {
+func (conn *Conn) querySQL(ctx *tcontext.Context, query string) (*sql.Rows, error) {
+	if conn == nil || conn.baseConn == nil {
 		return nil, terror.ErrDBUnExpect.Generate("database connection not valid")
 	}
 
-	var (
-		err  error
-		rows *sql.Rows
-	)
-
-	ctx.L().Debug("query statement", zap.String("sql", query), zap.Reflect("arguments", args))
 	startTime := time.Now()
+
+	params := retry.Params{
+		RetryCount:         10,
```

Comment:
I think set by the user is not properly, because users don't know real meaning of retryCount, it's usually face to internal errors such as `TiKVServerTooBusy`, if users set this retryCount too small or too big, it will bring some problems, let internal retry logic to deal with internal error is better.